app/models/role.rb

```diff
@@ -1,6 +1,9 @@
 class Role < ActiveRecord::Base
   belongs_to :resource, polymorphic: true
   has_and_belongs_to_many :users, join_table: :users_roles
+
+  has_paper_trail on: [:update], only: [:name, :description], meta: {conference_id: :resource_id}
```

Comment:
It is only tracking **update** action. Action **toggle_user** will also be tracked.